### PR TITLE
Token Geyser V2 UI

### PR DESCRIPTION
**Pull request by jx3yang**
## Description

- Added UI code in the `frontend/` folder
- Moved `sdk/` to `frontend/src/sdk`
- Changes related to subgraph and contract are documented in the `INFO.md` file
- Documentation relating to the frontend code can be found under `frontend/README.md`

### CODE REVIEW

File: test/Geyser.ts [hunk 1]
```diff
@@ -24,6 +24,16 @@ import {
...
 
+/*
+
+  Note: the implementation of increaseTime() was changed to only use `evm_mine` with the
+  timestamp passed as a parameter, instead of using a combination of `evm_increaseTime` and `evm_mine`
+
+  This implementation does not seem to run into the same problem as it previously did.
+  As a result, the indeterminancy issue is fixed
+
+*/
```
Comment (thegostep) [inline test/Geyser.ts:35]: 🤩

**Review by thegostep — APPROVED** [review]
reviewed the contract, sdk, and subgraph changes

**Comment by aalavandhan** [comment]
Hey @jx3yang can you resolve the conflict in `contracts/Geyser.sol` ? Once that's done we're good to merge 👍

**Comment by aalavandhan** [comment]
Thank you @jx3yang 👍

**Pull request merged**